test/model_factory_test.py

```diff
@@ -73,6 +85,12 @@ def setUp(self):
           model:
             cfg_type_name: a_type
             type: NamedModelElement
+        unused_type:
```

Comment:
name proposal: defined_but_unused_type - OR: add a comment stating that we define the type but decidedly do not define any elements